test/fixtures/customers.yml

```diff
@@ -0,0 +1,13 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```

Comment:
Some of the tests don't pass because the fixtures don't all have valid attributes. (e.g. phone number isn't a number)